Spanner/src/TypeAnnotationInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Spanner;
+
+/**
+ * @internal
+ * Contract for Spanner Values that need a typeAnnotationCode along with a type
+ * Note: This interface is internal and shouldn't be considered GA
+ */
+interface TypeAnnotationInterface
+{
+    /**
+     * @return int|string
```

Comment:
Looking at [`TypeAnnotationCode`](https://github.com/googleapis/google-cloud-php/blob/main/Spanner/src/V1/TypeAnnotationCode.php#L36) it looks like this should always return an int.

```suggestion
     * @return int
```